File: custom-addons/helpdesk_sale_timesheet/models/helpdesk_sla.py
# -*- coding: utf-8 -*-
# Part of Odoo. See LICENSE file for full copyright and licensing details.

from odoo import fields, models

class HelpdeskSLA(models.Model):
    _inherit = 'helpdesk.sla'

    sale_line_ids = fields.Many2many(
        'sale.order.line', string="Sales Order Items",
        domain="[('is_service', '=', True)]")

File: enterprise/helpdesk/static/tests/mock_server/mock_models/helpdesk_sla.js
import { fields, models } from "@web/../tests/web_test_helpers";

export class HelpdeskSla extends models.Model {
    _name = "helpdesk.sla";

    name = fields.Char();

    _records = [{ name: "SLA 1" }, { name: "SLA 2" }];
}
